README.md

```diff
@@ -9,9 +9,23 @@ npm
 I recommend using nvm
 
 ## Installation
+### Packages
 ```
 npm install
 ```
+### Database
+To create database run:
+```
+psql -c "CREATE DATABASE snippethero_development WITH ENCODING 'UTF8'
+```
+To setup app database configuration copy example file and overwrite it with your username and password to database:
+```
+cp server/config/config.json.example server/config/config.json
+```
+To run migrations
+```
+cd server; ../node_modules/.bin/sequelize db:migrate
```

Comment:
@Vakrim Can we add here npm task eg. ``` npm run migrate```

npm in its task loads by default local node_modules binaries, and that path to sequelize binary looks ugly
it will be enaugh if we add something like ``` "migrate": "sequalize -- db:migrate"```